lib/bullhorn/twilio.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Bullhorn.Twilio do
+  def deliver_sms_two_factor_token(user, token) do
+    ExTwilio.Message.create(
+      from: bullhorn_phone_number(),
+      message: "System76 Security Code: #{String.upcase(token)}",
+      to: user.phone_number
+    )
+  end
+
+  def deliver_voice_two_factor_token(user, token) do
+    spaced_out_token =
+      token
+      |> String.upcase()
+      |> String.split("")
+      |> Enum.join(" ")
```

Comment:
This is so the voice doesn't attempt to say the word and does so slowly. I'm going to test this a bit today. It may be even better to add a period after each.